instrumentation/spring/starters/spring-boot-starter/build.gradle.kts

```diff
@@ -5,12 +5,11 @@ plugins {
 
 group = "io.opentelemetry.instrumentation"
 
-val versions: Map<String, String> by project
-val springBootVersion = versions["org.springframework.boot"]
+val springBootVersion = "2.6.15"
 
 dependencies {
-  api("org.springframework.boot:spring-boot-starter:$springBootVersion")
-  api("org.springframework.boot:spring-boot-starter-aop:$springBootVersion")
+  compileOnly("org.springframework.boot:spring-boot-starter:$springBootVersion")
+  compileOnly("org.springframework.boot:spring-boot-starter-aop:$springBootVersion")
```

Comment:
why is this needed?

Reply:
the spring version was used in the tests - but the user should include the spring starter directly